tests/test_api_infrastructure.py

```diff
@@ -18,8 +18,9 @@
 import tempfile
 import unittest
 
+from aws_cdk import App
```

Comment:
I'd prefer to be explicit about `App`'s and `Stack`'s module, based on the same reasoning as in prior comments:
```suggestion
import aws_cdk as cdk
```